plugins/modules/remote_management/hpilo/hpilo_info.py

```diff
@@ -113,6 +113,13 @@
     returned: always
     type: str
     sample: 123456ABC78901D2
+
+host_power_status:
+    description: Power status of host
+    returned: always
+    type: str
+    sample: |
+      ON, OFF or UNKNOWN
```

Comment:
```suggestion
      ON
```
It does only return one of the values, not the string "ON, OFF or UNKNOWN" I think.

Reply:
The `|` operator in YAML makes it a multi-line string, which I supposed will output this as a string.

Reply:
`sample` should be an example of a value that is returned by the module. The module does not return the string "ON, OFF or UNKNOWN", but one of the strings "ON", "OFF" or "UNKNOWN". Thus `sample` should be either `ON`, `OFF` or `UNKNOWN`, but **not** the string `ON, OFF or UNKNOWN` as it is right now.